src/data-workspace/section-form/displayOptions.js

```diff
@@ -0,0 +1,25 @@
+const defaultDisplayOptions = {
+    beforeSectionText: '',
+    pivotMode: 'n/a',
+    afterSectionText: '',
+    pivotCategory: null,
+}
+export const getDisplayOptions = (section) => {
+    if (!section) {
+        return defaultDisplayOptions
+    }
+
+    try {
+        const { displayOptions: displayOptionString } = section
+
+        const displayOptions = JSON.parse(displayOptionString)
```

Comment:
I would've liked to validate the actual fields here, since it's an arbitrary string. But that would entail adding a validation lib, and a decision of what to use... Not sure if it's worth it since it seems we're checking for equality before using them. Except for `pivotedCategory` which could be used more defensively.